src/terminal/adapter/FontBuffer.cpp

```diff
@@ -222,9 +222,9 @@ gsl::span<const uint16_t> FontBuffer::GetBitPattern() const noexcept
     return { _buffer.data(), MAX_CHARS * _fullHeight };
 }
 
-til::size FontBuffer::GetCellSize() const
+til::size FontBuffer::GetCellSize() const noexcept
 {
-    return { _fullWidth, _fullHeight };
+    return { gsl::narrow_cast<til::CoordType>(_fullWidth), gsl::narrow_cast<til::CoordType>(_fullHeight) };

```

Comment:
This is weird... having to do the narrow_cast on the outside by digging into `til::CoordType`... we can't just accept this into the constructor and narrow it there especially if we're going to do `narrow_cast` (and not throw on error)?

Reply:
Oh, alright. I guess in context of the "whole story" then... it's fine. I'll let it pass. I keep remembering that you told me that all of this becomes less and less a problem the further we get to having actually unified types.